handleArray.js

```diff
@@ -0,0 +1,13 @@
+const handleObject = require('./handleObject.js')
```

Comment:
Perhaps for organization, it would be nice to put all the handleArray, handleObjects, handleString functions in a folder so that this project can be later scaled to include many more files

Reply:
I 100% agree for a production application.